### PR TITLE
docs: add pointer for Windows target platform

### PR DESCRIPTION
**Pull request by brodybits**
as discussed in: https://github.com/brodybits/create-react-native-module/issues/43#issuecomment-891320527

/cc @asklar

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,8 +38,7 @@ This tool based on [`react-native-create-library`](https://www.npmjs.com/package
       - issue [`#95`](https://github.com/brodybits/create-react-native-module/issues/95)
     - It is recommended to disable or remove Flipper in case of debug build issues - see issue [`#422`](https://github.com/brodybits/create-react-native-module/issues/422)
 - Out-of-tree target platforms
-  - Windows target platform - no longer supported for reasons discussed in issues [`#23`](https://github.com/brodybits/create-react-native-module/issues/23) and [`#43`](https://github.com/brodybits/create-react-native-module/issues/43). Existing Windows C# template is kept in `unsupported-platforms` for now (at least) and further discussion would be welcome in a new issue on GitHub.
-    NOTE that development on Windows as well as Linux *host* platofrms is supported for Android (only).
+  - Windows target platform - not supported directly, please use `react-native-windows-init` according to this procedure: https://microsoft.github.io/react-native-windows/docs/native-modules-setup#creating-a-new-native-module-library-project
```
Comment (asklar) [inline README.md:41]:
```suggestion
  - Windows target platform - not supported directly, please use `react-native-windows-init` according to this procedure: https://aka.ms/RNW-NativeModuleSetup
```

**Review by asklar — APPROVED** [review]
LGTM, thanks! - I created an aka.ms link so we can keep the target updated as we change the docs

**Comment by brodybits** [comment]
Much better with the aka.ms link, thanks.

I wish I could afford to give more time for better integrated Windows support.

**Comment by asklar** [comment]
thanks @brodybits !

**Pull request merged**